README.md

```diff
@@ -0,0 +1,88 @@
+# Direct Stability Parameters Change Module (DSPC)
+
+A module for the Sky Protocol that enables direct changes to stability parameters (duty, dsr, ssr) through a simple, secure interface with proper constraints and timelocks.
+
+## Overview
+
+The DSPC module provides a streamlined way to modify stability parameters in the Maker Protocol, including:
+- Stability fees (duty) for different collateral types via the Jug contract
+- Dai Savings Rate (DSR) via the Pot contract
+- Sky Savings Rate (SSR) via the sUSDS contract
+
+## Features
+
+- Batch updates for multiple rate changes
+- Two-level access control:
+  - Admins can configure the module
+  - Facilitators can propose and execute rate changes
+- Rate change constraints:
+  - Min/max caps per rate
+  - Max update delta
+- Event emission for all actions
+- Simple, auditable implementation
+
+## Installation
+
+```bash
+forge install
+```
+
+## Testing
+
+```bash
+forge test
+```
+
+## Usage
+
+1. Deploy the contract with the required addresses:
+```solidity
+DSPC dspc = new DSPC(
+    jugAddress,  // For stability fees
+    potAddress,  // For DSR
+    susdsAddress, // For SSR
+    convAddress  // For rate conversions
+);
+```
+
+2. Configure the module parameters:
+```solidity
+// Set timelock duration
+dspc.file("tau", 1 days);
+
+// Configure constraints for a collateral type
+dspc.file("ETH-A", "min", 1);     // Min rate: 0.01%
+dspc.file("ETH-A", "max", 1000);  // Max rate: 10%
```

Comment:
If this is the first configuration, it will fail because `min` would be greater than `max`.
The correct order to set in this case is `max > min > step`.